pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -722,15 +752,25 @@ public static Map<String, List<KitRequestShipping>> getAllKitRequestsByRealm(@No
         return kitRequests;
     }
 
+    public static Map<String, List<KitRequestShipping>> getAllKitRequestsByRealmForES(@NonNull String realm, boolean getAll) {
```

Comment:
Nit: I'm not a fan of long, inaccurate method names. Can we change this to `getAllKitRequestsForES`?